BitFaster.Caching/Lfu/CmSketchCore.cs

```diff
@@ -316,7 +311,7 @@ private unsafe int EstimateFrequencyAvx(T value)
                     .AsUInt16();
 
                 // set the zeroed high parts of the long value to ushort.Max
-#if NET6_0_OR_GREATER
+#if NET
                 count = Avx2.Blend(count, Vector128<ushort>.AllBitsSet, 0b10101010);
 #else
                 count = Avx2.Blend(count, Vector128.Create(ushort.MaxValue), 0b10101010);
```

Comment:
this line can be deleted - it is for net core 3.1 as mentioned earlier (earlier one is in the copy paste variant for the benchmark).